src/main/java/com/exactpro/th2/common/schema/message/impl/rabbitmq/parsed/RabbitParsedBatchSender.java

```diff
@@ -44,7 +48,17 @@ protected int extractCountFrom(MessageBatch message) {
 
     @Override
     protected byte[] valueToBytes(MessageBatch value) {
-        return value.toByteArray();
+        val groupBuilder = MessageGroup.newBuilder();
+
+        for (val message : value.getMessagesList()) {
+            val anyMessage = AnyMessage.newBuilder().setMessage(message).build();
+            groupBuilder.addMessages(anyMessage);
+        }
+
+        val batchBuilder = MessageGroupBatch.newBuilder();
+        val group = groupBuilder.build();
```

Comment:
Why don't use `var` instead? We compile the project for java 11 so this feature is available. I don't think that we should use the Lombok in this case